README.md

```diff
@@ -21,26 +22,27 @@ To store ToDo items, the project uses as Trello's REST API. To set it up:
 
 ## System Requirements
 
-The project uses poetry for Python to create an isolated environment and manage package dependencies. To prepare your system, ensure you have an official distribution of Python version 3.7+ and install poetry using one of the following commands (as instructed by the [poetry documentation](https://python-poetry.org/docs/#system-requirements)):
+### SSH
+Make sure you have created SSH keys and are able to use them to [connect to a remote machine](https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh)
 
-### Poetry installation (Bash)
+The project uses Vagrant to run the To Do app in a virtual machine that encapsulates the development environment in a single configuration file, making it easy to share and launch with a single command, `vagrant up`. To prepare your system, ensure you have an official distribution of Python version 3.7+. 
```

Comment:
Minor, but worth noting you've conflated these instructions somewhat - if you're using Vagrant then there's no need to install python since it'll be installed for us inside the Vagrant box!